pgjdbc/src/test/java/org/postgresql/jdbc/InsertRewriteWithAlternatingTypesIssue584.java

```diff
@@ -131,12 +134,12 @@ public void testPreparedStatementPreparation() throws Throwable {
         pst.executeBatch();
       } while (statementName == null && ++count < 10);
       Assert.assertEquals("S_1", statementName);
+      throw new RuntimeException("this is my exception!");
```

Comment:
It *was* intentional, but not intended to be left in. Thanks for pointing it out!